sycl/include/CL/sycl/detail/pi.h

```diff
@@ -691,6 +691,18 @@ pi_result piEnqueueKernelLaunch(
   const pi_event *  event_wait_list,
   pi_event *        event);
 
+pi_result piEnqueueNativeKernel(
+  pi_queue queue,
```

Comment:
Could you please use existing code style for function arguments (i.e. put start arg names from the same columns)?

Reply:
Done.